dydx/dydxPresenters/dydxPresenters/_v4/Onboarding/Tos/dydxTosViewBuilder.swift

```diff
@@ -40,9 +40,12 @@ private protocol dydxTosViewPresenterProtocol: HostedViewPresenterProtocol {
 }
 
 private class dydxTosViewPresenter: HostedViewPresenter<dydxTosViewModel>, dydxTosViewPresenterProtocol {
+    private let onboardingAnalytics = OnboardingAnalytics()
```

Comment:
consider init injection w/ default value for potentially testing in the future?

Reply:
Yeah, we should create the dependencies from the builder and pass it down 👍